Digital Signature/Verify-digital-sign-in-existing-PDF/.NET/Verify-digital-sign-in-existing-PDF/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
@sameerkhan001 , Change the sample name like. 

CheckIfPDFIsSigned

Reply:
Changed.